arches_controlled_lists/utils/skos.py

```diff
@@ -0,0 +1,204 @@
+                                allowed_languages[lang.code] = lang
+
+                        object_language = (
+                            allowed_languages[object.language] or default_lang
+                        )
+                        relation_or_value_type = predicate.replace(SKOS, "").replace(
+                            ARCHES, ""
+                        )
+                        list_item_value = ListItemValue(
+                            list_item=list_item,
+                            valuetype=skos_value_types.get(relation_or_value_type),
+                            language=object_language,
+                            value=self.unwrapJsonLiteral(str(object))["value"],
+                        )
+                        self.list_item_values.append(list_item_value)
+
+                    elif predicate == SKOS.broader:
+                        self.relations.append(
+                            {
+                                "source": self.generate_uuidv5_from_subject(
+                                    baseuuid, object
+                                ),
+                                "type": "broader",
+                                "target": list_item,
+                            }
+                        )
+                    elif predicate == SKOS.narrower:
+                        self.relations.append(
+                            {
+                                "source": list_item,
+                                "type": "narrower",
+                                "target": self.generate_uuidv5_from_subject(
+                                    baseuuid, object
+                                ),
+                            }
+                        )
+
+                    elif predicate == ARCHES.sortOrder:
+                        sortorder = int(self.unwrapJsonLiteral(str(object))["value"])
+
+                list_item.uri = uri
+                list_item.sortorder = sortorder
+                self.list_items[list_item_id] = list_item
+
+            ### Relationships ###
+            for relation in self.relations:
+                source = relation["source"]
+                target = relation["target"]
+                type = relation["type"]
+                if type == "narrower":
+                    self.list_items[target].parent = source
+                elif type == "broader":
+                    self.list_items[source].parent = target
+
+            with transaction.atomic():
+                List.objects.bulk_create(self.lists.values())
+                ListItem.objects.bulk_create(self.list_items.values())
+                ListItemValue.objects.bulk_create(self.list_item_values)
+
+                # TODO: Handle list item pk collisions in polyhierarhcies
+
+                ### Sort order ###
+                new_list_items = ListItem.objects.filter(
+                    list_id__in=[new_list.pk for new_list in self.lists.values()]
+                ).prefetch_related("children")
```

Comment:
Pretty minor detail, but you already have the created instance in the return value of bulk_create. Then you can still prefetch children with `prefetch_related_objects()`.